packages/core/types/src/modules/entity-service/params/attributes/relation.ts

```diff
@@ -11,7 +11,7 @@ interface PositionalArguments {
   end?: boolean;
 }
 
-type WithPositionArguments<T> = T & PositionalArguments;
+type WithPositionArguments<T> = T & { position: PositionalArguments };
```

Comment:
@Convly I missed this during the review, but the positional arguments of relations were not the right ones!

Do you think this is something I should add in another PR targetting to `main` ?

Reply:
Yep let's create a fix for `main` too, nice catch